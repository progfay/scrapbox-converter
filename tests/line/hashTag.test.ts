/* global describe it expect */

import { parse } from '@progfay/scrapbox-parser'
import convert from '../../src'
import pretty from 'pretty'

describe('hashTag', () => {
  it('Simple hashTag', () => {
    const input = `Hash Tag
#hashTag`
    const html = convert(parse(input), 'projectName')
    expect(pretty(html)).toMatchSnapshot()
  })
})
