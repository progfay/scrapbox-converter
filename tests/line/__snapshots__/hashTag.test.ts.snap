// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hashTag Simple hashTag 1`] = `
"<article>
  <h1 class=\\"title\\">Hash&nbsp;Tag</h1>
  <section>
    <p><a href=\\"hashTag\\">#hashTag</a></p>
  </section>
</article>"
`;
